_posts/2018-12-01-research-software.md

```diff
@@ -70,6 +70,7 @@ Here are some examples of research software registries:
 It will help others find and access the particular version of your software.
 Unique means that the identifier will point on and only version and location of your software.
 Persistent means that it will pointing to the same version and location for long, specified amount of time. For example Zenodo provides you with a DOI (Digital Object Identifier) that will be resolvable for at least the next 20 years.
+Recent initiatives as Software heritage propose to associate a permalinks as intrinsec SHA1 identifier to software (see example through the id: swh:1:dir:005bc6218c7a0a9ede654f9a177058adcde98a50 / permalinks: https://archive.softwareheritage.org/swh:1:dir:005bc6218c7a0a9ede654f9a177058adcde98a50/)
```

Comment:
"Recent initiatives as Software heritage" > "Recent initiatives, such as Software heritage,", intrinsec > intrinsic,

Reply:
done, thank you... and sorry for my approximate english...